microgen/mesh.py

```diff
@@ -143,17 +144,28 @@ def _iter_bounding_boxes(minimum: _Point3D, maximum: _Point3D, eps: float) -> It
 def _get_deltas(rve: Rve) -> np.ndarray:  # To add as a property of Rve?
     return np.array([rve.dx, rve.dy, rve.dz])
 
+def _get_min(rve: Rve) -> np.ndarray:  # To add as a property of Rve?
+    return np.array([rve.x_min, rve.y_min, rve.z_min])
+
+def _get_max(rve: Rve) -> np.ndarray:  # To add as a property of Rve?
+    return np.array([rve.x_max, rve.y_max, rve.z_max])
 
 def _iter_matching_bounding_boxes(rve: Rve, axis: int) -> Iterator[tuple[int, int]]:
+
     deltas = _get_deltas(rve)
     eps: float = 1.0e-3 * min(deltas)
-    minimum = np.zeros(_DIM_COUNT)
-    maximum = deltas
-    maximum[axis] = 0.
+    minimum = _get_min(rve)
+    maximum = _get_max(rve)
+    maximum[axis] = _get_min(rve)[axis]
```

Comment:
No need to create a new array:

```suggestion
    maximum[axis] = minimum[axis]
```

Reply:
This added been modified